app/components/InstructionCell.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { View, StyleSheet, Text } from 'react-native'
+import { colors } from '../config'
+
+const InstructionCell = ({ item }) => {
+  const {
+    mainView,
+    timeBgView,
+    timeTextView,
+    dayAndDateBgView,
+    dateView,
+    dayView,
+    sepratorViewStyle,
+  } = styles
+  const { sNo, title, body } = item
+
+  return (
+    <View>
+      <View style={mainView}>
+        <View style={timeBgView}>
+          <Text style={timeTextView}>{sNo}</Text>
+        </View>
+        <View style={dayAndDateBgView}>
+          <Text style={dateView}>{title}</Text>
+          <Text style={dayView}>{body}</Text>
+        </View>
+      </View>
+      <View style={sepratorViewStyle} />
```

Comment:
The Design doesn't have a separator, do you think it would look better with one? If so there's a divider component on another branch we can reuse here.

Reply:
here is already divider <View style={sepratorViewStyle} />

Reply:
<View style={sepratorViewStyle} />

Reply:
sepratorViewStyle

Reply:
I know its here, can we reuse the `<Divider /> from the develop branch? You'll need to merge but it will reduce duplication and ensure the dividers are consistent in the app.